rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/100_composite.yml

```diff
@@ -0,0 +1,154 @@
+setup:
+  - skip:
+      version: " - 8.0.99"
+      reason: introduced in 8.1.0
+
```

Comment:
Since it's a new field type, I'd recommend to also add unit tests in CompositeAggregatorTests for coverage.